docs/integrations/dapp-listing/defillama.md

```diff
@@ -143,7 +145,9 @@ module.exports = {
 
 Fetch adapters only allow us to get the TVL at the current time, so it's impossible to fill old values on a protocol's TVL chart or recompute them, thus leading to charts that look jumpy. To solve this we introduced SDK adapters, which allow us to retrieve a protocol's TVL at any point in time.
 
-> Fetch adapters can only be used for projects on non-EVM chains. Where possible, [SDK adapters](https://app.gitbook.com/o/-LgGrgOEDyFYjYWIb1DT/s/-M8GVK5H7hOsGnYqg-7q-872737601/integration/dapp-listing/defillama) are preferred to fetch adapters because on-chain calls are more transparent.
+:::caution
+Fetch adapters can only be used for projects on non-EVM chains. Where possible, [SDK adapters](https://app.gitbook.com/o/-LgGrgOEDyFYjYWIb1DT/s/-M8GVK5H7hOsGnYqg-7q-872737601/integration/dapp-listing/defillama) are preferred to fetch adapters because on-chain calls are more transparent.
```

Comment:
Please link the SDK adapters to: https://docs.llama.fi/list-your-project/how-to-write-a-fetch-adapter

Reply:
updated.